src/util/worker/mod.rs

```diff
@@ -303,6 +332,23 @@ impl<T: Display + Send + 'static> Worker<T> {
     pub fn start<R>(&mut self, runner: R) -> Result<(), io::Error>
     where
         R: BatchRunnable<T> + Send + 'static,
+    {
+        let timer: Option<Timer<EmptyTask>> = None;
+        let f = |_: &mut Timer<EmptyTask>, _: TimeoutTask<EmptyTask>| {};
+        self.start_with_timer(runner, timer, f)
+    }
+
+    /// Start the worker.
+    pub fn start_with_timer<R, U, F>(
+        &mut self,
+        runner: R,
+        timer: Option<Timer<U>>,
+        f: F,
```

Comment:
There should be an `on_timeout` method in the definition of `Runner`, why pass a closure?